openslides/users/static/js/users/site.js

```diff
@@ -529,9 +531,18 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
             if (!user.groups) {
                 user.groups = [];
             }
+            // inject the changed user (copy) object back into DS store
+            User.inject(user);
+            // save change user object on server
             User.save(user).then(
                 function(success) {
                     $scope.closeThisDialog();
+                },
+                function(error) {
+                    // save error: revert all changes by restore
+                    // (refresh) original user object from server
+                    User.refresh(user);
+                    $scope.closeThisDialog();
```

Comment:
Right, we should show the error in the dialgo and NOT close the dialog... I'll fix it.